ptvsd/socket.py

```diff
@@ -45,6 +61,80 @@ def ignored_errno(*ignored):
             raise
 
 
+class KeepAlive(namedtuple('KeepAlive', 'interval idle, maxfails')):
```

Comment:
Does the comma in the field list have any significance?

Reply:
It's an innocuous typo. :)  I'll fix it.